src/SecTester.Bus/Internal/SnakeCaseNamingPolicy.cs

```diff
@@ -0,0 +1,14 @@
+using System.Text.Json;
+using SecTester.Core.Utils;
+
+namespace SecTester.Core.Json;

```

Comment:
Please make sure namespaces are aligned with folders
```suggestion
namespace SecTester.Bus.Internal;

```